api/Schema/Mutations/EmployeeScheduleMutation.cs

```diff
@@ -0,0 +1,28 @@
+using api.Context;
+using api.Requests;
+using api.Services;
+using Microsoft.EntityFrameworkCore;
+
+namespace api.Schema.Mutations
+{
+    [ExtendObjectType("Mutation")]
+    public class EmployeeScheduleMutation
+    {
+        public async Task<string> CreateEmployeeSchedule([Service] IDbContextFactory<HrisContext> contextFactory, [Service] EmployeeScheduleService _employeeSchedueleService, CreateEmployeeScheduleRequest request)
+        {
+            using HrisContext context = contextFactory.CreateDbContext();
+            try
+            {
+                using var transaction = context.Database.BeginTransaction();
+                var createEmployeeSchedule = await _employeeSchedueleService.Create(request);
+
+                transaction.Commit();
+                return createEmployeeSchedule;
+            }
+            catch (GraphQLException)
+            {
+                throw;
+            }
```

Comment:
Can you try this code. Please let know if it works.


```suggestion
            catch (GraphQLException ex)
            {
                throw new GraphQLException("Error creating employee schedule", ex);
            }
```

Reply:
It is now currently being addressed. Pero iba yung method, pero same lang na nag rereturn ng custom error message. I'll let the gc know when once its done po 🙇